static/app/views/eventsV2/results.tsx

```diff
@@ -579,6 +579,13 @@ const Top = styled(Layout.Main)`
   flex-grow: 0;
 `;
 
+const Middle = styled(Layout.Main)`
+  @media (min-width: ${p => p.theme.breakpoints[1]}) and (max-width: ${p =>
```

Comment:
In `static/app/views/eventsV2/resultsChart.tsx` at line 289 it defines a breakpoint to remove margin from the ResultsChart:
```js
const StyledPanel = styled(Panel)`
  @media (min-width: ${p => p.theme.breakpoints[1]}) {
    margin: 0;
  }
`;
```

I was able to bump the min-width breakpoint to `breakpoints[2]` and get the same behaviour, should we make that smaller change there? Seems like we only need the margin under `breakpoints[2]` because after that it uses CSS grid

Reply:
Ah nice, that's probably a much simpler fix. I've updated to this instead, thanks!